index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="RU">
+<head>
+	<meta charset="UTF-8">
```

Comment:
В остальных файлах используются пробелы, а здесь табы. Лучше, чтобы везде были пробелы.

```suggestion
    <meta charset="UTF-8">
```